README.md

```diff
@@ -1009,19 +1009,22 @@ perform or avoid the evaluation of the askokcancel function.
   These are deprecated and we've found bugs in them.   Don't use them!   Use
   subprocess instead.
 
-### Don't use subprocess.Popen with a string argument (#508) ###
+### Don't use subprocess.Popen with a string argument ###
+(See SeattleTestbed/attic#508)
 
-  Call subprocess.Popen with a first argument that looks like ['command',
-  'arg1', 'arg2', ...].   Do not use 'command arg1 arg2 ...'!
+  Call subprocess.Popen with a first argument that looks like `['command',
+  'arg1', 'arg2', ...]`.   Do not use `'command arg1 arg2 ...'`!
 
 
-### Avoid changing directory (#487) ###
+### Avoid changing directory ###
+(See SeattleTestbed/attic#487)
```

Comment:
Should this be a link?  `SeatleTestbed` is a different GitHub organization (this one's `secure-systems-lab`).